src/app/layout/footer/footer.component.ts

```diff
@@ -1,11 +1,28 @@
 import { Component } from '@angular/core';
+import { environment } from 'src/environments/environment';
 
 @Component({
   selector: 'gp-footer',
   templateUrl: './footer.component.html',
   styleUrls: ['./footer.component.scss']
 })
 export class FooterComponent {
+
+  sponsors = [
+    {
+      logoUrl: environment.firebaseStorageUrl +
+        'images%2Fsponsors%2Ffundacion-jala-logo-white.png?alt=media&token=5bc5b4d9-6815-4e75-ba2f-812dd381d2f3',
```

Comment:
I think we don't need the token in the path because the rules of our storage allow to read data without auth.